protos/dcs/mission/v0/mission.proto

```diff
@@ -303,6 +303,20 @@ message StreamEventsResponse {
     dcs.common.v0.Airbase place = 3;
   }
 
+  // Occurs when a chat message is sent on the server
+  message PlayerSentChatEvent {
+    // The player's id in the current server session.
+    uint32 playerId = 1;
+    // what was typed
+    string message = 2;
+    // to whom?
+    MessageTarget target = 3;
+    enum MessageTarget {
+      COALITION = 0;
```

Comment:
@rurounijones you know if the protobuf linter would also complain here that the `0` entry its not named `UNSPECIFIED`?

Reply:
Good catch!. Yes same issue and same resolution. The enum field names also need to contain the enum name as follows:

```proto
enum MessageTarget {
  MESSAGE_TARGET_UNSPECIFIED = 0
  MESSAGE_TARGET_COALITION = 1
  MESSAGE_TARGET_ALL = 2
}
```

Then + 1 in the lua with a comment.

Reply:
Addressed in https://github.com/DCS-gRPC/rust-server/pull/89/commits/0e60ac92fb4bc2fd8f3e558c3e6efbef4532e4b9